tests/foreman/cli/test_contentaccess.py

```diff
@@ -15,251 +17,153 @@
+                'host': vm.hostname,
+                'packages-restrict-applicable': 'true',
+                'search': f'name={REAL_RHEL7_0_2_PACKAGE_NAME}',
             }
         )
-        # Publish the content view
-        ContentView.publish({'id': content_view['id']})
-        cls.content_view = ContentView.info({'id': content_view['id']})
-
-    def _setup_virtual_machine(self, vm):
-        """Make the initial virtual machine setup
-
-        :param VirtualMachine vm: The virtual machine setup
-        """
-        setup_virtual_machine(
-            vm,
-            self.org['label'],
-            rh_repos_id=[repo['repository-id'] for repo in self.repos if repo['cdn']],
-            repos_label=self.repos_info[1][0]['label'],
-            lce=ENVIRONMENT,
-            patch_os_release_distro=DISTRO_RHEL7,
-            install_katello_agent=True,
-        )
+        if applicable_packages:
+            break
+        time.sleep(10)
+    assert len(applicable_packages) > 0
+    assert REAL_RHEL7_0_2_PACKAGE_FILENAME in [
+        package['filename'] for package in applicable_packages
+    ]
+
+
+@pytest.mark.tier2
+@pytest.mark.upgrade
+def test_positive_erratum_installable(vm):
+    """Ensure erratum applicability is showing properly, without attaching
+    any subscription.
+
+    :id: e8dc52b9-884b-40d7-9244-680b5a736cf7
+
+    :steps:
+        1. register a host to unrestricted org with Library
+        2. install a package, that will need errata to be applied
+        3. list the host applicable errata with searching the required
+            errata id
+
+    :expectedresults: errata listed successfuly and is installable
+
+    :BZ: 1344049, 1498158
+
+    :CaseImportance: Critical
+    """
+    # check that package errata is applicable
+    for _ in range(30):
+        erratum = Host.errata_list({'host': vm.hostname, 'search': f'id = {REAL_0_ERRATA_ID}'})
+        if erratum:
+            break
+        time.sleep(10)
+    assert len(erratum) == 1
+    assert erratum[0]['installable'] == 'true'
+
+
+@pytest.mark.tier2
+def test_negative_rct_not_shows_golden_ticket_enabled():
+    """Assert restricted manifest has no Golden Ticket enabled .
+
+    :id: 754c1be7-468e-4795-bcf9-258a38f3418b
+
+    :steps:
+
+        1. Run `rct cat-manifest /tmp/restricted_manifest.zip`.
+
+
+    :expectedresults:
+        1. Assert `Content Access Mode: Simple Content Access` is not present.
+
+    :CaseImportance: High
+    """
+    # need a clean org for a new manifest
+    org = entities.Organization().create()
+    # upload organization manifest with org environment access disabled
+    manifest = manifests.clone()
+    manifests.upload_manifest_locked(org.id, manifest, interface=manifests.INTERFACE_CLI)
+    result = ssh.command(f'rct cat-manifest {manifest.filename}')
+    assert result.return_code == 0
+    assert 'Content Access Mode: Simple Content Access' not in '\n'.join(result.stdout)
```

Comment:
no need to join on a line break here. could just be joined by `''` if a join is required at all.

Reply:
OK, with `''.join` it works, but not without join. Will fix.

Reply:
Done